dist/hidapi.podspec

```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |spec|
+
+  spec.name         = "hidapi"
+  spec.version      = "0.9.0"
+  spec.summary      = "A Simple library for communicating with USB and Bluetooth HID devices on Linux, Mac and Windows."
+
+  spec.description  = <<-DESC
+  HIDAPI is a multi-platform library which allows an application to interface with USB and Bluetooth HID-Class devices on Windows, Linux, FreeBSD, and macOS. HIDAPI can be either built as a shared library (.so, .dll or .dylib) or can be embedded directly into a target application by adding a single source file (per platform) and a single header.
+                   DESC
+
+  spec.homepage     = "https://github.com/libusb/hidapi"
+
+  spec.license      = { :type=> "GNU GPLv3 or BSD or HIDAPI original", :file => "LICENSE.txt" }
+
+  spec.authors      = { "Alan Ott" => "alan@signal11.us",
+                        "Ludovic Rousseau" => "rousseau@debian.org",
+                        "libusb/hidapi Team" => "https://github.com/libusb/hidapi/graphs/contributors", 
+                        "Vitaly Samoylik" => "vitalys@gmail.com" }
```

Comment:
As official CocoaPods [documentation](https://guides.cocoapods.org/syntax/podspec.html#authors) states, `spec.authors` has to be a project author/maintainers only.
OSS community (so as you @vitalys) makes a great contribution into this project development, but still, contributors are not project authors neither maintainers.

> https://github.com/libusb/hidapi/graphs/contributors

this link shows all the contributors, not the libusb/hidapi Team.
libusb/hidapi Team is a set of people who have direct write access to the repository. Unfortunately, there is no publicly available list of people, who are part of this team. As an alternative, I'd suggest to give a link to AUTHORS.txt: https://github.com/libusb/hidapi/blob/master/AUTHORS.txt.

Reply:
Fixed